client/src/components/Conversations/Convo.tsx

```diff
@@ -26,16 +23,17 @@ export default function Conversation({ conversation, retainView, toggleNav, isLa
   const { showToast } = useToastContext();
 
   const { conversationId, title } = conversation;
-  const inputRef = useRef<HTMLInputElement | null>(null);
+  const inputRef = useRef(null);
   const [titleInput, setTitleInput] = useState(title);
   const [renaming, setRenaming] = useState(false);
 
-  const clickHandler = async (event: React.MouseEvent<HTMLAnchorElement>) => {
-    if (event.button === 0 && event.ctrlKey) {
-      toggleNav();
-      return;
+  useEffect(() => {
```

Comment:
is it necessary to use \`useEffect\` here??

Reply:
shouldn't the focus cover this? In my experience, useEffect should be the last resort, and there could be a better way.

Reply:
fixed in: d26c2c16be6779806a776c93c2601e3a21c59021